packages/neuron-ui/package.json

```diff
@@ -71,33 +72,42 @@
     "@storybook/preset-create-react-app": "4.1.2",
     "@storybook/react": "6.5.12",
     "@storybook/testing-library": "0.0.13",
+    "@storybook/addon-knobs": "5.3.21",
+    "@storybook/addon-storyshots": "5.3.18",
+    "@storybook/addon-viewport": "5.3.18",
+    "@storybook/addons": "5.3.18",
+    "@storybook/cli": "5.3.18",
```

Comment:
These `@storybook`'s plugins should be removed. `@storybook` has been updated to 6.5.12 by https://github.com/nervosnetwork/neuron/pull/2506/files#diff-0563a7b3e27fdf2d3e488aa5c618a241c5ba435a435c120acfa80a357a307a20